Orbit V3 (#1815)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md

> As part of the activity of an Orbit chain, different fees are collected with every transaction. These fees are collected as a single amount (the transaction fees) but are internally split into different components depending on their purpose. Each component can also be transferred to a different fee collector address that can be configured on your chain.

Reviewer: ```suggestion
Different fees get collected for every transaction as part of an Orbit chain activity. These fees are collected as a single amount (the transaction fees) but split internally into different components depending on their purpose. Each component is transferrable to a different fee collector address that is configurable on your chain.
```

> This guide describes the different fees that are collected on your chain, how to configure them, and how to specify the fee collector address for each fee type.

Reviewer: ```suggestion
This guide describes the different collected fees and explains how to specify the fee collector address on your chain for each fee type.
This guide describes the different fees collected on your chain, how to configure them, and how to specify the fee collector address for each type.
```

- [L2 fees](/how-arbitrum-works/09-gas-fees.mdx#l2-gas-pricing) for the Orbit base fee and surplus fee
- [L1 fees](/how-arbitrum-works/09-gas-fees.mdx#l1-gas-pricing) for the Parent chain base fee and surplus fee

## How to configure the fees collected?

> Let's see in what ways we can configure each fee type.

Reviewer: ```suggestion
Let's see in what ways we can configure each fee type:
```

### Orbit (minimum) base fee

Your chain is configured with a minimum base fee for execution. This value can be obtained by calling the method `getMinimumGasPrice()(uint256)` of the [ArbGasInfo](/build-decentralized-apps/precompiles/02-reference.mdx#arbgasinfo) precompile.

```shell
cast call --rpc-url $ORBIT_CHAIN_RPC 0x000000000000000000000000000000000000006C "getMinimumGasPrice() (uint256)"
```

Alternatively, you can use the Orbit SDK to retrieve the minimum Orbit base fee configured:

```typescript
const orbitChainClient = createPublicClient({
    chain: <OrbitChainDefinition>,
    transport: http(),
}).extend(arbGasInfoPublicActions);

const orbitMinimumBaseFee = await orbitChainClient.arbGasInfoReadContract({
    functionName: 'getMinimumGasPrice',
});
```

> _Note: This minimum base fee defines the minimum value that the chain's base fee can have. However, in periods of congestion, the actual base fee might be higher than this minimum. Check the next section "Orbit surplus fee" for more information._

Reviewer: ```suggestion
::: note
This minimum base fee defines the minimum value that the chain's base fee can have. However, in periods of congestion, the actual base fee might be higher than this minimum. Check the next section "Orbit surplus fee" for more information.
:::
```
